tests/test_on_slave.py

```diff
@@ -101,6 +101,7 @@ def test_is_open_xapi_exception(self, mock_util):
         """
         Exceptions from is_open are logged
         """
+        import on_slave
```

Comment:
Imports at top of file only

Reply:
The line is to fix UT coverage 99% issue which complains [test_on_slave.py#L26](https://github.com/xapi-project/sm/blob/1ef66556a41f70e9fa9e4621d08857c1e45cc6e2/tests/test_on_slave.py#L26) is missing. Put it on the top of file cannot take effect as the mock. side_effect is configured here [self.mock_import.side_effect = self.fake_import](https://github.com/xapi-project/sm/blob/1ef66556a41f70e9fa9e4621d08857c1e45cc6e2/tests/test_on_slave.py#L36C45-L36C56). We have to put import on_slave after it.

Reply:
14c35ee3320fc0f888fefab92135aab1213ff154 fixes the problem properly. Please review, squash and repush.

Reply:
Thanks. The PR is updated with this fix.